src/api-reference/request/v4.get-started.markdown

```diff
@@ -15,131 +13,88 @@ Version 4.0 of Request API works only with the new [Authentication API](/api-ref
+This API supports both company level and user level access tokens.
 
-Examples:
+#### Company Access Token
 
-cURL
+* The `userId` parameter is required to provide the user identity of who made the API call.
+* Does not have an associated role.
 
-```shell
-curl -d "client_secret={YOUR SECRET}&client_id={YOUR CLIENT ID}&grant_type=password&username={YOUR USERNAME}&password={YOUR PASSWORD}" https://us.api.concursolutions.com/oauth2/v0/token
-```
+#### User Access Token
 
-HTTPie
+* The `userId` parameter is not required.
+* Requires the Web Service Admin role to call the API.
 
-```shell
-http -f POST https://us.api.concursolutions.com/oauth2/v0/token client_secret={YOUR SECRET} client_id={YOUR CLIENT ID} grant_type=password username={YOUR USERNAME} password=P{YOUR PASSWORD}
-```
+## <a name="explore-api"></a>Explore the API
 
-### <a name="javascript">Explore the API in JavaScript
-
-Below are some simple NodeJS code snippets for getting a token and issuing a request to the API.
-
-#### <a name="retrieve-token"></a>Retrieve a User or Company Access Token:
-
-```javascript
-'use strict';
-const request = require('request');
-request.post({
-  url:'https://us.api.concursolutions.com/oauth2/v0/token',
-  form: {
-    client_secret: 'YOUR VALUE HERE',
-    client_id: 'YOUR VALUE HERE',
-    username: 'YOUR VALUE HERE',
-    password: 'YOUR VALUE HERE',
-    grant_type: 'password'
-  }},
-  (err, httpResponse, body) => {
-    if(err) { console.log(err); }
-    console.log('Status:', httpResponse.statusCode);
-    console.log('Response:', body);
-  });
-```
+### <a name="prerequisites"></a>Prerequisites
+
+Read the [Getting Started](/api-reference/authentication/getting-started.html) section of [Authentication API](/api-reference/authentication/apidoc.html).
 
-#### <a name="post-request"></a>Post a request:
-
-```javascript
-'use strict';
-const https = require('https');
-const ACCESS_TOKEN = 'YOUR ACCESS TOKEN HERE';
-const USER_ID = 'YOUR VALUE HERE';
-const request = JSON.stringify(YOUR_REQUEST_HERE);
-const options = {
-  hostname: 'us.api.concursolutions.com',
-  path: `/travelrequest/v4/users/${USER_ID}`,
-  method: 'POST',
-  headers: {
-    'Authorization': `Bearer ${ACCESS_TOKEN}`,
-    'Content-Type': 'application/json',
-    'Content-Length': Buffer.byteLength(request)
-  }
-};
-const req = https.request(options, (res) => {
-  console.log('statusCode:', res.statusCode);
-  console.log('headers:', res.headers);
-
-  res.on('data', (data) => {
-    process.stdout.write(data);
-  });
-});
-
-req.write(request);
-req.end();
-
-req.on('error', (e) => {
-  console.error(e);
-});
+[Run in Postman](https://app.getpostman.com/run-collection/bfe85f4a4e435a161a8a)
```

Comment:
Add this Postman collection link to the main table of contents for this page.

Reply:
@laurentlhermine  it is already the case in the file in the TVR, it has not been taken into account?

Reply:
done